bentoml/_internal/server/instruments.py

```diff
@@ -96,25 +134,158 @@ async def wrapped_send(message: "ext.ASGIMessage") -> None:
                 status_code = message["status"]
```

Comment:
Let's change the behavior for api_server, too.

Reply:
ooops